src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlDataReader.cs

```diff
@@ -1720,7 +1728,7 @@ virtual public XmlReader GetXmlReader(int i)
                 if (_data[i].IsNull)
                 {
                     // A 'null' stream
-                    return SqlTypeWorkarounds.SqlXmlCreateSqlXmlReader(new MemoryStream(new byte[0], writable: false), closeInput: true, async: false);
+                    return SqlTypeWorkarounds.SqlXmlCreateSqlXmlReader(new MemoryStream(Array.Empty<byte>(), writable: false), closeInput: true, async: false);
```

Comment:
Could we just use `Stream.Null` here?

Reply:
I don't think we can - `Stream.Null` doesn't do anything when written to, while the returned MemoryStream will have `CanWrite` set to `false`.

Reply:
Ah, good point!